about.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: About
+permalink: /about/
+---
+
+This reference is currently a work in progress. Not all pages may be complete and pages may be updated at any time without notice.
+
+If you're knowledgeable about payments and have something contribute, your additions would be much appreciated! Additionally any feedback is also welcome. Please see [contributions](/contributing/) for how to contribute and provide feedback.
```

Comment:
Should it be something like "Please see the [contributing page]..." to match the page title?